lib/perl/OVH/Bastion/log.inc

```diff
@@ -100,7 +100,9 @@ sub syslogFormatted {
 }
 
 sub warn_syslog {
-    my $msg = shift;
+    my $msg   = shift;
+    my $noisy = shift;
```

Comment:
I don't see this used in the other files this PR embeds?

Reply:
It is, at the very end of the `load_configuration()` func of `configuration.inc`